data/src/main/java/org/akvo/flow/data/net/FlowApiService.java

```diff
@@ -20,6 +20,16 @@
 
 package org.akvo.flow.data.net;
 
-public interface FlowApiService {
+import org.akvo.flow.data.entity.ApiLocaleResult;
 
+import retrofit2.http.GET;
+import retrofit2.http.Headers;
+import retrofit2.http.Url;
+import rx.Observable;
+
+interface FlowApiService {
+
+    @GET
```

Comment:
I have tried everything and it just does not work. The way we build the url is: baseUrl + surveyedlocale? + all the query params  + &h=the query prams encoded with HMacSha1, which is not something that can be easily done with retrofit. Also the params have to be ordered alphabetically so if they are reordered it will not work either.